src/main/java/bookmarked/Book.java

```diff
@@ -36,6 +36,15 @@ public void borrowBook(LocalDate borrowDate, Period borrowPeriod) {
         }
     }
 
+    public void extendDueDate(int extraDays) {
+        if (this.returnDate != null) {
+            this.returnDate = this.returnDate.plusDays(extraDays);
```

Comment:
I think better consistent? this one can extend by any days but in ExtendCommand is fixed by 7 days (a week)